nixos/modules/programs/yazi.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
Can you try to remove this? There is an effort to remove such wide-scoped `with` statements.

Reply:
Fixed.